samples/security/spm/spm_nspe/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _spm_psoc6:
+
+SPM base NS environment example for PSoC6  CM4
+###########
+
+Overview
+********
+A simple SPM base NS environment example for PSoC6  CM4.
+Needs Secure part on CM0+
+
+CM4(spm_nspe) sends debug log to Uart Port5 which is available through USB connection with the PSoC6 kit.
+CM0+(spm_spe) send debug messages to UART port 12(pin0(RX)/pin1(TX)) � connect external UART to UASB convertor to these pins on kit 
+
+Project description:
+- CM0p starts CM4 core
+- CM4 writes the message with the counter in loop to the shared memory block, triggers interrupt to CM0p core and wait on response.
+- Once interrupt received, CM0p prints out received message and initiate response interrupt to CM4.
+
+Building and Running
+********************
+cmake -G"Eclipse CDT4 - Ninja" -DBOARD=cy8ckit_062_wifi_bt_m0 ..
```

Comment:
script and code examples need to be included in a code-block directive with the body of the code block indented by three spaces:

    .. code-block: bash

       cmake ...